vision/use-cases/brochure-creation-using-imagen/brochure_creation_using_imagen.ipynb

```diff
@@ -222,43 +214,47 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 69,
+      "execution_count": null,
       "id": "ea4c7f0f-14c0-4b8b-b466-c9d8133cf640",
       "metadata": {
-        "id": "79735e0572f1"
+        "id": "8c9b4e340c4b"
       },
       "outputs": [],
       "source": [
         "from enum import Enum\n",
         "import os\n",
         "import random\n",
         "import time\n",
+        "import warnings\n",
         "\n",
         "from IPython import display\n",
         "from PIL import Image\n",
         "from PIL import Image as PIL_Image\n",
         "from PIL import ImageColor\n",
+        "from google.cloud import translate\n",
```

Comment:
Since there are multiple versions (v2, v3) you should specify which version to use, then alias.

```py
from google.cloud import translate_v3 as translate
```